extensions/oidc-common/runtime/src/main/java/io/quarkus/oidc/common/runtime/OidcCommonConfig.java

```diff
@@ -232,6 +232,12 @@ public void setMethod(Method method) {
             @ConfigItem(defaultValue = "password")
             public String keyPassword;
 
+            /**
+             * The token keyId, added to the JWT header during token exchange. This is required by some IDPs.
```

Comment:
Can you please change it to `Key identifier of the signing key added as a JWT 'kid' header` ? (sorry, I initially suggested a `The token keyid`).
Note this is not related to the token exchange (which is possible during the token propagation) and it is not really related to the process of the code to token exchange as such - it is only one of the options of how a client can authenticate, in this case it is JWT authentication. This is used by both `quarkus-oidc` and `quarkus-oidc-client`

Reply:
Javadoc & commit message modified as requested